src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/JDBCEncryptionDecryptionTest.java

```diff
@@ -568,12 +569,12 @@ private void testGetObject(ResultSet rs,
                         && objectValue3.equalsIgnoreCase("" + values[index]);
 
                 if (("" + values[index]).length() >= 1000) {
-                    assertTrue(matches, "\nDecryption failed with getObject() at index: " + i + ", " + (i + 1) + ", " + (i + 2)
-                            + ".\nExpected Value at index: " + index);
+                    assertTrue(matches, TestResource.getResource("R_decryptionFailed") + "getObject(): " + i + ", " + (i + 1) + ", " + (i + 2)
+                            + ".\n" + TestResource.getResource("R_expectedValue") + index);
```

Comment:
`R_expectedValue `is "Expected Value:", lets have another String here for "Expected Value at index:"